src/app/admin-panel/admin-panel/students/students.component.scss

```diff
@@ -32,7 +32,7 @@
 }
 .studentsForm ::ng-deep input
 {
-  width: 100%;
+  width: 100% !important;
```

Comment:
why is this in the ng-deep?

Reply:
input must be acceptable

Reply:
why i use !important, or what?

Reply:
Removed !importent.

Reply:
the question was: why do you use ng-deep if this input is the component template?

Reply:
without ng-deep, i can't set style for p-inputMask and p-calendar